Abhi-AnimeX/app/src/main/java/com/abhijith/animex/ui/screens/animedetails/AnimeDetailsUiState.kt

```diff
@@ -0,0 +1,9 @@
+package com.abhijith.animex.ui.screens.animedetails
+
+import com.abhijith.animex.domain.model.AnimeItem
+
+sealed class AnimeDetailsUiState {
+    data object Loading : AnimeDetailsUiState()
```

Comment:
why data object, why not simple object?

Reply:
to be very frank Android studio suggested changing it. 
THat said i had a quick look through and the reason can be to keep consistency with the other data classes inside the sealed class and using a data object can help in auto generating toString, hashnode and equals methods @anandwana001